pkg/pillar/tgt/tgt.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package tgt
```

Comment:
Can you add a comment describing what this package does?

Reply:
Added